lib/strategies/webapp-strategy.js

```diff
@@ -344,12 +345,18 @@ function getTokens(req, options, strategy) {
 		code: req.query.code
 	};
 
+	function cb(err, IDToken, info) {
+		return err ? strategy.fail(err) : strategy.success(IDToken || null, info);
```

Comment:
There is also a strategy.error() function we should use in here. We should use error if err exists, fail if IDToken is null, and success otherwise